### ISSUE TITLE
Update dependencies, remove vulnerabilities

### ISSUE THREAD

**Issue by ciampo** [ciampo/macro-carousel#41]
Update npm dependencies (like focus visible, inert, polyfills..). This should be also an opportunity to get rid of vulnerabilities
